src/util/slackAlert.ts

```diff
@@ -0,0 +1,13 @@
+import fetch from 'node-fetch'
+
+import { syncedSettings } from '../db/couchSettings'
+
+export function slackAlert(text: string): void {
+  const { slackUri } = syncedSettings.doc
+
```

Comment:
Will this post multiple duplicated messages to Slack? In other words, will the problem we solved [here](https://app.asana.com/0/1190649553204286/1202424870074259/f) happen again in push server?

Reply:
It could happen. However, I think the rates server is poorly designed. The server should *never* throw exceptions. Everything we expect to go wrong should have an appropriate `catch` handler to return a real client error type, like 404, 401, or whatever. This Slack logging is only for "OMG WTF" moments where something truly unexpected has taken place. If you need to throttle these, you haven't put enough "normal" error handling into the app.